tasks/request_status_for_job/request_status_for_job.py

```diff
@@ -214,6 +215,14 @@ def handler(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
         Or, if an error occurs, see create_http_error_dict
             400 if asyncOperationId is missing. 500 if an error occurs when querying the database.
     """
+    # get the secret ARN from the env variable
+    try:
+        secret_arn = os.environ["SECRET_ARN"]
+    except KeyError as key_error:
+        LOGGER.error(
+            "SECRET_ARN environment value not found."
+        )
+        raise key_error
```

Comment:
You can just `raise` without specifying the error. I don't know if it matters in Python, but in some languages doing `raise exception` resets the stack trace. Recommend simplifying here and elsewhere.

Reply:
removed from here and other places